components/SearchBar/SearchBar.tsx

```diff
@@ -133,7 +133,7 @@ const SearchBar = ({ big }: SearchProps) => {
   function search(word?: string, event?) {
     setNextUserState({ numOfSearches: Number(userState.numOfSearches) + 1 })
 
-    if ((word && word !== '') || searchValue !== '') {
+    if ((word && word !== '') || !!searchValue) {
```

Comment:
You could further simplify this line without change of functionality.
```suggestion
    if (word || searchValue) {
```

Reply:
:+1:  make sense, thanks